task-loops/uninstall.sh

```diff
@@ -0,0 +1,11 @@
+kubectl delete Service -n tekton-pipelines -l app.kubernetes.io/part-of=tekton-experimental-task-loops
```

Comment:
Could this script be replaced with `kubectl delete -f config/` ?

It could be nice to isolate looping task components in their own namespace like the CEL and Wait task controllers are, so it's easier to add/remove them without having to touch other things. But all of that's feedback for future work and doesn't have to happen in this PR.

Reply:
Yes. Fixed.